terraform/shared/shared.tf.json.template.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+from azul import (
+    config,
+)
+from azul.terraform import (
+    emit_tf,
+)
+
+trail_names = {
+    'dev': 'Default',
+    'prod': 'platform-hca-cloudtrail'
+}
+
+emit_tf({
+    'data': {
+    },
+    'resource': {
+        'aws_s3_bucket': {
```

Comment:
How come you don't need to import this resource. It's a pre-existing bucket right?